From c163fb8f6af875013a89f2a86734855ad090571d Mon Sep 17 00:00:00 2001
Message-Id: <c163fb8f6af875013a89f2a86734855ad090571d.1421709247.git.chang-joon.lee@intel.com>
In-Reply-To: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
References: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
From: Shobhit Kumar <shobhit.kumar@intel.com>
Date: Thu, 25 Sep 2014 15:23:55 +0530
Subject: [PATCH 18/75] SQUASHME! [VPG]: drm/i915: Fix compilation issues with
 i915 sync fence disabled.

This is a miss in forklift. Patch should be reverted in next forklift.

commit 979afd5830c0ad6ff4fff216860b187617e2b027
Author: John Harrison <John.C.Harrison@Intel.com>
Date:   Tue Apr 1 17:25:21 2014 +0100

    REVERTME [VPG]: drm/i915: Refactored native sync code for better file splits

For: GMINL-3547
Change-Id: Id3e913fbc935af27de3f92aa11d336532a38ca14
Signed-off-by: Shobhit Kumar <shobhit.kumar@intel.com>
---
 drivers/gpu/drm/i915/intel_sync.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_sync.h b/drivers/gpu/drm/i915/intel_sync.h
index 51fe6b2..123d5a84 100644
--- a/drivers/gpu/drm/i915/intel_sync.h
+++ b/drivers/gpu/drm/i915/intel_sync.h
@@ -97,7 +97,7 @@ void i915_sync_reset_timelines(struct drm_i915_private *dev_priv)
 }
 
 static int i915_sync_create_fence(struct intel_engine_cs *ring, u32 seqno,
-				  int *fd_out)
+				  int *fd_out, u64 ring_mask)
 {
 	return 0;
 }
-- 
1.7.9.5

